<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>neue Kurseinheit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="rb_new">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ein neuer „Kurseinheit“ soll angelegt werden. Das bedeutet entweder eine neue Unterrichtsstunde oder eine Deputatsangabe ohne Unterrichtsstunden.&lt;/p&gt;&lt;p&gt;Die Stunde kann Teil eines Blocks sein (einer Zeit, in der mehrere Kurse parallel oder Epochenmäßig stattfinden).&lt;/p&gt;&lt;p&gt;Für jeden Kurs darf nur einmal eine Einheit der jeweiligen Sorte (einfache Stunde, Blockstunde oder Deputatsstunden ohne Unterricht) angelegt werden. Weitere Unterrichtsstunden können dann mit der „+“-Schaltfläche hinzugefügt werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>neu</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">choose_group</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb_add2block">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Der „Kurs“ soll zu einem schon existierenden Block hinzugefügt werden.&lt;/p&gt;&lt;p&gt;Zwischen zeitbegrenzten Blöcken („Epochen“) und parallelen, durchlaufenden Unterrichtsstunden wird hier nicht differenziert.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>zu Block hinzufügen</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">choose_group</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb_add2team">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diese Option ist für Teamteaching oder Klassen- bzw. Gruppenübergreifenden Unterricht gedacht.&lt;/p&gt;&lt;p&gt;Der aktuelle „Kurs“ soll mit einem anderen kombiniert werden. Das Fach der Kurse muss gleich sein, sie teilen einen Raum und die Deputatsangaben müssen für alle gleich sein.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Team/Gruppe</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">choose_group</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_block">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;„Block“ bedeutet hier, dass Unterrichtseinheiten entweder parallel (zur gleichen Zeit) oder nacheinander („Epochenunterricht“) stattfinden.&lt;/p&gt;&lt;p&gt;Zwischen zeitbegrenzten Blöcken („Epochen“) und parallelen, durchlaufenden Unterrichtsstunden wird hier nicht differenziert.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Block / „Epoche“</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="blockstack">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page_block">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Blockfach:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="block_subject"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Block-Kennzeichen:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="BLOCK_TAG">
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="insertPolicy">
                <enum>QComboBox::NoInsert</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_noblock">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QRadioButton" name="rb_simple">
               <property name="text">
                <string>einfache Stunde</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb_onlypay">
               <property name="text">
                <string>nur Deputat</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Unterrichtsstunden:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list_lessons"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Blockkurse:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="table_courses">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>80</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Klasse</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Gruppe</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fach</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Lehrkraft</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Raum</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>10G</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>R</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>Mathematik</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>Hans Bockmühl</string>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string>RS+</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="choose_group"/>
 </buttongroups>
</ui>
